### PR TITLE
feat: E2E tests for dashboard

### PR DESCRIPTION
**Pull request by harishv7**
## Problem

Lacking E2E test for dashboard

Closes IS-260

## Solution

Adds E2E tests for dashboard

**Breaking Changes**

- [ ] Yes - this PR contains breaking changes
  - Details ...
- [X] No - this PR is backwards compatible

## Test

Ensure that `dashboard.spec.ts` E2E test passes successfully

### CODE REVIEW

File: cypress/e2e/dashboard.spec.ts [hunk 1]
```diff
@@ -18,35 +30,82 @@ describe("dashboard flow", () => {
...
+      .should("be.visible")
+      .should("have.attr", "href", ISOMER_GUIDE_LINK)
+      .should("have.attr", "target", "_blank")
   })
-  it.skip("should navigate to the settings page when manage site settings is clicked", () => {
-    throw new Error("Not implemented")
+
+  it("should navigate to the settings page when manage site settings is clicked", () => {
+    // Act
+    console.log(TEST_REPO_NAME)
```
Comment (seaerchin) [inline cypress/e2e/dashboard.spec.ts:79]: could we remove this?

File: cypress/utils/dashboard.ts [hunk 2]
```diff
@@ -1,6 +1,11 @@
 import { E2E_EMAIL_TEST_SITE } from "../fixtures/constants"
 import { COMMENTS_DRAWER_BUTTON_SELECTOR } from "../fixtures/selectors/dashboard"
 
+// NOTE: This is a `div` tag styled like a `button`\
```
Comment (seaerchin) [inline cypress/utils/dashboard.ts:4]: nit: remove the `\`

**Review by seaerchin — APPROVED** [review]
lgtm w minor nits

File: cypress/utils/dashboard.ts (hunk 2 above)
Comment (harishv7) [reply cypress/utils/dashboard.ts]: done

File: cypress/e2e/dashboard.spec.ts (hunk 1 above)
Comment (harishv7) [reply cypress/e2e/dashboard.spec.ts]: done

**Pull request merged**